src/Commands/AdminCommand.php

```diff
@@ -123,6 +123,13 @@ protected function getUser($create = false)
 
         // If we need to create a new user go ahead and create it
         if ($create) {
+            // check if user with given email exists
+            $User = $model::where('email', $email)->first();
```

Comment:
You should move this check later on because at this point you might still not have the email.

Reply:
I moved it further down, after the check for the email.